src/sal.js

```diff
@@ -137,10 +137,10 @@ const onIntersection = (entries, observer) => {
     if (entry.intersectionRatio >= options.threshold) {
       animate(entry);
 
-      if (options.once) {
+      if (options.once && entry.target.dataset.salRepeat === undefined) {
```

Comment:
Please use assign `entry.target.dataset.salRepeat !== undefined` to a constant. Something like:

```js
/**
 * IntersectionObserver callback
 * @param  {Array<IntersectionObserverEntry>} entries
 * @param  {IntersectionObserver} observer
 */
const onIntersection = (entries, observer) => {
  entries.forEach((entry) => {
    const { target } = entry;
    const hasRepeatFlag = target.dataset.salRepeat !== undefined;

    if (entry.intersectionRatio >= options.threshold) {
      animate(entry);

      if (options.once && !hasRepeatFlag) {
        observer.unobserve(target);
      }
    } else if (!options.once || hasRepeatFlag) {
      reverse(entry);
    }
  });
};
```